main.cpp

```diff
@@ -4,30 +4,35 @@
 #include <sstream>
 #include <vector>
 #include <algorithm>
+#include <random>
 
 namespace
 {
-int random_no(unsigned int range)
+size_t randomRange(size_t min, size_t max)
 {
-	srand(time(NULL));
-	int secret_num = rand() % range;
-	return secret_num;
+	static std::random_device device;
+	static std::default_random_engine engine(device());
+	std::uniform_int_distribution<> dist(min, max);
+	return dist(engine);
 }
 
-std::string to_lowercase(std::string_view word)
+std::string to_lowercase(std::string_view const word)
 {
 	std::string lowercased = word.begin();
```

Comment:
FYI, you can just pass the view to the constructor as well: `std::string lowercased(word)`